api/v1/amazoncloudintegration_types.go

```diff
@@ -61,6 +61,11 @@ type AmazonCloudIntegrationSpec struct {
 type AmazonCloudIntegrationStatus struct {
 	// State provides user-readable status.
 	State string `json:"state,omitempty"`
+
+	// Conditions represents the latest observed set of conditions for the component. A component may be one or more of
+	// Ready, Progressing, Degraded or other customer types
```

Comment:
This note applies to all of the `*_types.go`

With the current implementation, I believe it should be simply:
```
// Conditions represents the latest observed set of conditions for the component. A component may be one or more of
// Ready, Progressing, Degraded.
```
(Please note that all comments should end with a period.)

Reply:
Done